### DISCUSSION TITLE
Sticky Advanced Settings?

### DISCUSSION THREAD

**Discussion by Explorographer** [lllyasviel/Fooocus#992] [Ideas]
Yeah, would just like the choices I make to stick.

**Comment by kubilaykilinc** [comment]
+1

**Comment by Sven-Lyco** [comment]
You can create your own preset if you want to reuse it frequently.

**Reply by kubilaykilinc** [reply]
Yes we can. But not for all parameters. yet

**Reply by Sven-Lyco** [reply]
Which parameters are not supported in the preset? I think in the presets in the json files there are a lot of possibilities to set in advance

**Reply by kubilaykilinc** [reply]
I couldn't make "Developer debug mode" checked by default. I can't set Freeu to checked - and I couldn't change its settings as default.

**Comment by Explorographer** [comment]
Yes, but who wants to write a preset?  People use Fooocus for simplicity...not so they can write their own .Json files.  Also, things like persistent Prompts and Negative prompts would be nice.  A save state so if you have to restart or the power goes out, you don't have to re-type/set everything.  Just an idea.  I am loving the rest of the program.

**Reply by kubilaykilinc** [reply]
I think writing presets is a very good approach to prepare different setups for different workflows. However, it would be nice to have the ability to save default settings.

**Reply by Explorographer** [reply]
I run an Automatic1111 install as well as a ComfyUI setup with a Discord Bot.  I have all the script/preset tweaking I need.  I thought the philosophy behind Foooocus was to get the tweaker out of the way of the creative.  That is what I think needs to be applied to this problem.  I know there are ways to get it done but I am looking to spend more time creating and less time tweaking.  The exact reason why Fooocus has become my go to for image generation these days.   Sorry if I am out of line here.  I will just leave this be. Take the advice or not, it's all good.
